src/applications/vaos/containers/ContactInfoPage.jsx

```diff
@@ -34,35 +34,57 @@ const initialSchema = {
         },
       },
     },
+    email: {
+      type: 'string',
+      format: 'email',
+    },
   },
 };
 
 const uiSchema = {
-  'ui:title': 'Where can we call to confirm your appointment?',
-  'ui:description':
-    'A scheduling clerk will contact you to coordinate an appointment date',
+  'ui:description': (
+    <>
+      <p>
+        This is the contact information we have on file for you. We’ll use this
+        information to contact you about scheduling your appointment. You can
+        update your contact information here, but the updates will only apply to
+        this tool.
+      </p>
+      <p className="vads-u-margin-y--2">
+        If you want to update your contact information for all your VA accounts,
+        please{' '}
+        <a href="/profile" target="_blank" rel="noopener noreferrer">
+          go to your profile page
+        </a>
+        .
+      </p>
+    </>
+  ),
   phoneNumber: phoneUI('Phone number'),
   bestTimeToCall: {
-    'ui:title': 'Best times for VA to call',
+    'ui:title': 'Best times for us to call you',
     morning: {
-      'ui:title': 'Morning (8 a.m. to noon EST)',
+      'ui:title': 'Morning (8 a.m. to noon)',
       'ui:options': {
         widgetClassNames: 'vaos-form__checkbox',
       },
     },
     afternoon: {
-      'ui:title': 'Afternoon (noon - 4 p.m. EST)',
+      'ui:title': 'Afternoon (noon - 4 p.m.)',
       'ui:options': {
         widgetClassNames: 'vaos-form__checkbox',
       },
     },
     evening: {
-      'ui:title': 'Evening (4 p.m. to 8 p.m. EST)',
```

Comment:
Interesting. Is it communicated that all times are eastern? Or are they?

Reply:
They're local times, but we're supposed to show the timezone. I'm going to leave that to a later ticket and am making this change so it isn't misleading in the meantime.

Reply:
They're local times, but we're supposed to show the timezone. I'm going to leave that to a later ticket and am making this change so it isn't misleading in the meantime.